data/scenarios/00-classic.yaml

```diff
@@ -0,0 +1,20 @@
+name: Classic game
+description: The classic game.  Foo bar.
+robots:
+  - name: base
+    loc: [0,0]
+    dir: [1,0]
+    display:
+      char: Ω
+      attr: robot
+    devices:
+      - 3D printer
+      - dictionary
+      - grabber
+      - life support system
+      - logger
+      - solar panel
+      - workbench
```

Comment:
I think we should *not* add that, because once we do #35 there will be no need to have default devices at all any more.  Robots will get exactly what devices they need to execute, if the base has them in stock.

Reply:
Right, that is better, I noticed the Issue only after making the suggestion. :smile: